### ISSUE TITLE
Feedback Aufgabe 05

### ISSUE THREAD

**Issue by MelvinBusch** [LMKodalle/EIA2#3]
1. Konzept und AD's in eine Datei bündeln und dann mit einem Link im Steckbrief versehen.
2. AD's sind zu ungenau. Siehe z.B. `append fieldset to body`, wo wird denn das fieldset-Element erzeugt? Oder `set Atrributes for Radio-Buttons`, woher weiß das Programm von wo die Attribute kommen? Bei solchen Dingen solltest du etwas spezifischer im AD sein, da sie entscheidend für das Verständnis des Programmablaufes sind.
3. Input-Knoten gehören nicht in die Mitte eines AD's. Interaktionen des Nutzers sind unvorhersehbar und lassen sich deshalb nicht mitten in einen linearen Programmablauf einbeziehen.
4. Im AD wird auf einen Funktion `click()` verwiesen, hierzu gibt es aber keine AD's...
5. `displayEis1` und `displayEis2` sind keine besonders aussagekräftigen Funktionsnamen. Was passiert hier genau? Immer treffende Namen verwenden.

Ansonsten alles in Ordnung 👍🏻

**Comment by LMKodalle** [comment]
2. Ich weiß nicht genau was du meinst. Ein Fieldset element wird dem body als Kind angehängt und die Attribute werden bei den verschiedenen Switch cases den jeweiligen input elementen hinzugefügt. Ich habe mich an Beispielen orientiert und so wurde es dargestellt.
3. Welcher Input-Knoten? Ich glaube du meinst den eventlistener? Wie soll ich das denn Darstellen wenn ein eventlistener geaddet wird. Ich habe mich an dem Beispiel im Heftchen für ADs orientiert und so wurde es dargestellt.
4. Click() war eine Funktion aus der letzten Aufgabe und wurde in dem zugehörigen Konzept behandelt.

**Comment by Plagiatus** [comment]
2. in displayEis1 steht als allererstes "append fieldset to body". Ich gehe davon aus, dass er sich darauf bezieht.
3. da musst du aber nochmal genau hinschauen, weil so ist das da nicht drin.

**Comment by Plagiatus** [comment]
Was nicht ganz passt ist dass mir in der Gesamtübersicht nur das unterste ausgewählte Topping angezeigt wird. Der Preis wird aber korrekt berechnet.

**Comment by LMKodalle** [comment]
Okay jetzt verstehe ich, es fehlt noch ein create fieldset davor. Das mit den Attributen habe ich aber noch nicht ganz verstanden? Wie soll ich da genauer drauf eingehen und bei dem Eventlistener finde ich im Internet keine hilfreichen Beispiele. Soll ich statt einem Eventkasten einfach ein Aktiviätsknoten mit ( add Eventlistener for click ) oder so machen?

**Comment by Plagiatus** [comment]
genau, einfach einen Aktivitätsknoten mit "addEventListener" machen.
Da gehe ich aber sowieso morgen nochmal darauf ein.
